tests/test_client.py

```diff
@@ -330,6 +344,23 @@ def test_upload_raster(identity_key):
     assert len(responses.calls) == 4
 
 
+@pytest.mark.parametrize('type', ('xyz', 'wms'))
+@responses.activate
+def test_upload_remote_raster(type):
+    client = _client()
+    add_mock_remote_raster_upload_responses(type)
+    add_mock_operations_responses('processing')
+    add_mock_operations_responses('success')
+    footprint = {
+        "type": "Polygon",
+        "coordinates": [ [ [10.0, 0.0], [11.0, 0.0], [11.0, 1.0],  [10.0, 1.0], [10.0, 0.0] ] ]
+    }
+    with pytest.raises(ValueError):
+        client.upload_remote_raster('spam', 'http://example.com', 0.2, footprint)
+    client.upload_remote_raster(type, 'http://example.com', 0.2, footprint)
+    assert len(responses.calls) == 3
```

Comment:
Perhaps just document here what the 3 expected calls are.

Reply:
Checked